lib/analytics-page/addon/application/styles.scss

```diff
@@ -15,6 +15,11 @@
     margin-top: 20px;
 }
 
+:global(.modal-body) {
```

Comment:
Instinctively, the application of this global class makes me nervous, but it's just limited to the analytics engine it seems and it makes sure any additional BsModal doesn't suffer from the no-scroll behavior... I'm torn, but I think it's good.